test/manifest_test.py

```diff
@@ -608,4 +608,8 @@ def test_cover_type_on_store(self):
 
     def test_get_protocol_preview(self):
         preview = get_protocol_preview(self.protocol, "TestMethod", manifest="test/manifest_test.json")
-        assert isinstance(preview, dict)
+        assert "my_string" in preview
```

Comment:
very minor nit, but could you accumulate these keys into a list and iterate through that instead?

Reply:
Definitely!
So something like:
```
manifest_keys = ["my_string", "my_container", "my_volume", "my_length", "my_bool"]
for key in manifest_keys:
    assert (key in preview)
```

Reply:
yup, something like that would work

Reply:
great! thanks!